### PR TITLE
Add changelog of v1.9.1

### PR DESCRIPTION
**Pull request by lunny**
As title

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,32 @@ This changelog goes through all the changes that have been made in each release
...
+  * Use .ExpiredUnix.IsZero to display green color of forever valid gpg key (#7850) (#7846)
+  * Do not fetch all refs (#7797) (#7837)
+  * Fix duplicate call of webhook (#7824) (#7821)
+  * Enable switching to a different source branch when PR already exists (#7823)
+  * Rewrite existing repo units if setting is not included in api body (#7811)
+  * Prevent Commit Status and Message From Overflowing On Branch Page (#7800) (#7808)
+  * API: fix multiple bugs with statuses endpoints (Backport #7785) (#7807)
+  * Fix Slack webhook fork message (1.9 release backport) (#7783)
+  * Fix approvals counting (#7757) (#7777)
+  * fix rename failed when rewrite public keys (#7761) (#7769)
```
Comment (lafriks) [inline CHANGELOG.md]:
```suggestion
  * Fix rename failed when rewrite public keys (#7761) (#7769)
```

**Review by zeripath — APPROVED** [review]

**Review by sapk — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Comment by lafriks** [comment]
Btw does drone uses latest golang 1.12.8 as it contains security fixes in std lib that we could be also affected

**Comment by lafriks** [comment]
Seems like it is using latest golang so we should probably mention this also in changlog under security section?

**Comment by lunny** [comment]
@lafriks done.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,33 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.9.1](https://github.com/go-gitea/gitea/releases/tag/v1.9.1) - 2019-08-14
+* BREAKING
+  * Add pagination for admin api get orgs and fix only list public orgs bug (#7742) (#7752)
+* SECURITY
+  * Be more strict with git arguments (#7715) (#7762)
+  * Releases builded with go 1.12.8 to fix security fixes in golang std lib, ref: https://groups.google.com/forum/#!topic/golang-nuts/fCQWxqxP8aA
```
Comment (lafriks) [inline CHANGELOG.md]:
```suggestion
  * Release built with go 1.12.8 to fix security fixes in golang std lib, ref: https://groups.google.com/forum/#!topic/golang-nuts/fCQWxqxP8aA
```

**Comment by lafriks** [comment]
Make LG-TM work :)

**Pull request merged**